pkg/console/subresource/route/route.go

```diff
@@ -206,21 +207,19 @@ func GetCustomTLS(customCertSecret *corev1.Secret) (*CustomTLSCert, error) {
 	if !certExist {
 		return nil, fmt.Errorf("custom cert secret data doesn't contain 'tls.crt' entry")
 	}
-	certificateVerifyErr := certificateVerifier(cert)
-	if certificateVerifyErr != nil {
-		return nil, fmt.Errorf("failed to verify custom certificate PEM: " + certificateVerifyErr.Error())
-	}
-	customTLS.Certificate = string(cert)
-
 	key, keyExist := customCertSecret.Data["tls.key"]
 	if !keyExist {
 		return nil, fmt.Errorf("custom cert secret data doesn't contain 'tls.key' entry")
 	}
-
 	privateKeyVerifyErr := privateKeyVerifier(key)
 	if privateKeyVerifyErr != nil {
 		return nil, fmt.Errorf("failed to verify custom key PEM: " + privateKeyVerifyErr.Error())
 	}
+	certificateVerifyErr := certificateVerifier(cert)
```

Comment:
I think you may want to keep this as is and move it back to where you're checking for certificate existence.

Reply:
moved back 👍